frontend/src/components/terminal/Terminal.tsx

```diff
@@ -4,9 +4,13 @@ import { useTerminal } from "../../hooks/useTerminal";
 
 import "@xterm/xterm/css/xterm.css";
 
-function Terminal() {
+interface TerminalProps {
+  secrets: string[];
+}
+
+function Terminal({ secrets }: TerminalProps) {
```

Comment:
**Suggestion:** Consider using default props for `secrets` to enhance component usability.

**Existing Code:**
```TypeScript
function Terminal({ secrets }: TerminalProps) {
```

**Improved Code:**
```TypeScript
function Terminal({ secrets = [] }: TerminalProps) {
```

**Details:**
Set default values for the `secrets` prop to ensure the component behaves correctly even if no secrets are passed.